crates/stc_ts_type_checker/tests/tsc.rs

```diff
@@ -891,7 +921,9 @@ fn libs_with_deps(libs: &[Lib]) -> Vec<Lib> {
             | Lib::Scripthost
             | Lib::WebworkerGenerated
             | Lib::WebworkerImportscripts
-            | Lib::WebworkerIterableGenerated => {}
+            | Lib::WebworkerIterableGenerated
+            | Lib::Decorators
+            | Lib::DecoratorsLegacy => {}
```

Comment:
I didn't understand where to write these.
Referenced by [es5.d.ts](https://github.com/microsoft/TypeScript/blob/83069ef10d5b9c8686f0a821955dd3fd7b6119a0/src/lib/es5.d.ts#L1-L2)

Reply:
I think it's fine to figure it out in follow-up PRs